src/ssl.c

```diff
@@ -21305,6 +21502,28 @@ int wolfSSL_RAND_bytes(unsigned char* buf, int num)
     return ret;
 }
 
+
+int wolfSSL_RAND_poll()
+{
+    WOLFSSL_ENTER("wolfSSL_RAND_poll");
+    byte  entropy[16];
+    int  ret = 0;
+    const int entropy_sz = 16;
```

Comment:
Can you try changing "const int" to "word32" for the type of "entropy_sz" variable?  The 3rd parameter of wc_GenerateSeed() expects a "word32".  I believe this is what is causing the Windows Visual Studio test to fail.

Reply:
Thank you for suggestion. But I still got errors.  Yes, Jenkins errors is not helpful. I will make sure on my windows machine again.

Reply:
I checked on my Visual Studio 2017 environment and could build with no error.
I also ran testsuites and got no error.